src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -379,6 +391,17 @@ private static void addConnectorConfigs(ConfigDef configDef) {
             ++order,
             Width.SHORT,
             BATCH_SIZE_DISPLAY
+        ).define(
+            MAX_BULK_SIZE_BYTES_CONFIG,
+            Type.LONG,
+            MAX_BULK_SIZE_BYTES_DEFAULT,
+            between(-1, 10 * MEGABYTES_TO_BYTES),
```

Comment:
have you done some research on the max? looks like ES can do up to `Integer.MAX_VALUE`

Reply:
(Resolved in 1:1) I think you're right. According to the [ES doc](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/modules-http.html), `http.max_content_length` will be set to the default if the user tries anything larger than `Integer.MAX_VALUE`.

I'll update the max, thanks!